scalafix/output/src/main/scala/migrate/RscCompat.scala

```diff
@@ -12,7 +12,7 @@ object RscCompat {
   }
 
   class Patterns {
-    val List() = List.apply[Nothing]()
+    val List() = List()
```

Comment:
Tried to debug this but it seems the bug comes from semantidb or Scalafix. There is no `TypeApplyTree` here anymore.